src/cli/commands/dht/find-peer.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const print = require('../../utils').print
+
+module.exports = {
+  command: 'findpeer <peerID>',
+
+  describe: 'Find the multiaddresses associated with a Peer ID.',
+
+  builder: {},
+
+  handler ({ ipfs, peerID, resolve }) {
+    resolve((async () => {
+      const peers = await ipfs.dht.findPeer(peerID)
+      const addresses = peers.multiaddrs.toArray().map((ma) => ma.toString())
+
+      print(addresses)
```

Comment:
`print` is just `console.log` which will give you output like:

```js
[ '/ip4/127.0.0.1/tcp/4001',
  '/ip4/169.254.83.88/tcp/4001',
  '/ip4/192.168.1.95/tcp/4001' ]
```

In comparison go-ipfs outputs:

```console
$ ipfs dht findpeer QmZMxNdpMkewiVZLMRxaNxUeZpDUb34pWjZ1kZvsd16Zic
/ip6/2001:19f0:5:66d1::/tcp/4001
/ip4/149.28.228.162/tcp/33154
/ip4/45.77.150.82/tcp/4001
/ip6/::1/tcp/4001
/ip6/2001:19f0:5:5d40:5400:1ff:fec1:ce94/tcp/4001
/ip4/127.0.0.1/tcp/4001
/ip4/149.28.228.162/tcp/4001
/ip4/127.0.0.1/tcp/8081/ws
```